TaskHero/ParseClient.swift

```diff
@@ -123,6 +123,66 @@ class ParseClient: NSObject {
         }
     }
     
+    func deleteTask(task: Task, success: @escaping () -> (), failure: @escaping (Error) -> ()) {
+        let query = PFQuery(className: "Task")
+        query.order(byAscending: "createdAt")
```

Comment:
Oh, sorry, I meant only this line: "query.order(byAscending: "createdAt")" might not be necessary since you're pulling a single object. This method is useful!

Sorry for the confusion :)